Automattic/wp-calypso
# Fix DotPager page positioning issue due element focus

#### Changes proposed in this Pull Request

This PR fixes the issue of `DotPager`'s current page positioning being changed as an effect of changing focus (e.g.: via keyboard tabbing) to an element in the next page. 

The proposed solution is to set `visibility: hidden` to the previous and next pages in `DotPager`, this makes the elements within those pages not focusable.

#### Testing instructions



* Head to My Home.
* In one of the `DotPager`, click on the current page and press tab until the last focusable element is focused.
* Press tab again, and the next focused element should not change the `DotPager`'s current page positioning. 

![Screen Shot 2022-02-18 at 1 37 16 PM](https://user-images.githubusercontent.com/797888/154624333-84898160-d3e5-4dd7-9f7f-b13036cf8283.png)




Related to #53617
